File: app/src/main/java/de/tum/in/tumcampusapp/models/tumo/Identity.java
package de.tum.in.tumcampusapp.models.tumo;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Root(name = "row")
public class Identity {
    @Element
    private String vorname = "";

    @Element
    private String familienname = "";

    @Element
    public String kennung = "";

    @Override
    public String toString() {
        return vorname + " " + familienname;
    }
}

File: app/src/main/java/de/tum/in/tumcampusapp/models/tumo/Identity.kt
package de.tum.`in`.tumcampusapp.models.tumo

import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

@Root(name = "row")
data class Identity(@field:Element var vorname: String,
                    @field:Element var familienname: String,
                    @field:Element var kennung: String,
                    @field:Element var obfuscatd_id: String,
                    @field:Element var obfuscatd_ids: ObfuscatedIds) {
    override fun toString(): String = "$vorname $familienname"
}
